src/JavascriptVM/JavascriptVM.tsx

```diff
@@ -102,24 +102,24 @@ export const VMProvider: FunctionComponent = ({ children }) => {
   const store = useStore();
 
   // handler for robot handlers, all calls to setMotorPower will update state in redux
-  const robot_handler: ProxyHandler<Handles.RobotHandle> = {
-    get: function (target, property, receiver) {
-      if (property === "setMotorPower") {
-        const originalImpl = target[property];
-        return function (channel: number, power: number) {
-          dispatch(
-            robotSimulatorSlice.actions.setPower({
-              channel: channel,
-              power: power,
-            })
-          );
-          return originalImpl.apply(target, [channel, power]);
-        };
-      }
-
-      return Reflect.get(target, property, receiver);
-    },
-  };
+  // const robot_handler: ProxyHandler<Handles.RobotHandle> = {
+  //   get: function (target, property, receiver) {
+  //     if (property === "setMotorPower") {
+  //       const originalImpl = target[property];
+  //       return function (channel: number, power: number) {
+  //         dispatch(
+  //           robotSimulatorSlice.actions.setPower({
+  //             channel: channel,
+  //             power: power,
+  //           })
+  //         );
+  //         return originalImpl.apply(target, [channel, power]);
+  //       };
+  //     }
+
+  //     return Reflect.get(target, property, receiver);
+  //   },
+  // };
```

Comment:
we can remove these lines if they are not required anymore

Reply:
done